content/sensu-go/5.0/guides/troubleshooting.md

```diff
@@ -69,6 +69,28 @@ following those logs are described. The name of the desired service, e.g.
 _NOTE: Platform versions described above are for reference only and do not
 supercede the documented [supported platforms][platforms]._
 
+### Log Messages
+
+#### Permission Issues
+
+Files and folders within `/var/cache/sensu/` need to be owned by the sensu user and group. You will see a logged error similar to the following if there is a permission issue with either the sensu-backend or the sensu-agent:
+
+{{< highlight json >}}
+{"component":"agent","error":"open /var/cache/sensu/sensu-agent/assets.db: permission denied","level":"fatal","msg":"error executing sensu-agent","time":"2019-02-21T22:01:04Z"}
+{{< /highlight >}}
+
+You can use a recursive `chown` to resolve permission issues with the sensu-backend:
+
+{{< highlight shell >}}
+sudo chown -R sensu:sensu /var/cache/sensu/sensu-backend
+{{< /highlight >}}
+
+or the sensu-agent:
+
+{{< highlight shell >}}
+sudo chown -R sensu:sensu /var/cache/sensu/sensu-agent
```

Comment:
I'm getting a `chown: invalid user: ‘sensu:sensu’` error here.

Reply:
I am not getting that error in sensu go stand alone or in the sandbox. Error would indicate that it can't find the `sensu` user. Can you run ` cat /etc/passwd | grep sensu` if your environment is still around? I get:

```
 cat /etc/passwd | grep sensu
sensu:x:994:991:Sensu Monitoring Framework:/opt/sensu:/bin/false
```

Reply:
I encountered this error using the Sensu binaries, which is expected. Marking this as resolved.